samples/apps/bookinfo/src/details/Dockerfile.sidecar

```diff
@@ -0,0 +1,37 @@
+# Copyright 2017 Istio Authors
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+FROM ruby:2.3
+
+RUN mkdir /opt/microservices
+RUN mkdir /opt/istio
+RUN mkdir /etc/istio
+RUN mkdir /etc/istio/proxy/
+RUN apt-get update && apt-get -y install iptables curl
+RUN adduser -disabled-password --gecos "" --uid 1337 istio
```

Comment:
we called this user istio-proxy in the proxy docker:
https://github.com/istio/proxy/blob/master/docker/Dockerfile.debug#L29

Reply:
thanks, resolved.